setup.py

```diff
@@ -28,13 +28,13 @@
     ),
     python_requires=">=3.8, <4",
     install_requires=[
-        "aiohttp[speedups]==3.8.1",
+        "aiohttp[speedups]==3.8.4",
```

Comment:
> aiohttp[speedups]==3.8.4 
aiohttp[speedups]>=3.8.4

---

unless we want to manually do updates every major release

Reply:
Same comment as above. It might be worth looking into but generally we won't benefit from taking the latest version of libraries unless they fix an issue we have